### PR TITLE
feat: added alert component, refactor: changed to good CSS practices

### PR DESCRIPTION
**Pull request by paarthbajaj**

### CODE REVIEW

File: alert.html [hunk 1]
```diff
@@ -0,0 +1,128 @@
...
+          <li><a href="./images.html">Images</a></li>
+          <li><a href="./input.html">Input</a></li>
+        </ul>
+      </aside>
+      <hr class="vertical-line" />
+      <main class="grow-1">
+        <h2 class="heading-1">Alerts</h2>
+        <p>
+          Provide contextual feedback messages for typical user actions with the
+          handful of available and flexible alert messages.<br />Alerts are
```
Comment (rkapoor10) [inline alert.html:40]: you may use 2 p tags instead of break tags

**Review by rkapoor10 — APPROVED** [review]
Code is well structured; good to merge.

File: alert.html (hunk 1 above)
Comment (paarthbajaj) [reply alert.html:40]: May I know the reason behind that?

**Review by mariyasada — APPROVED** [review]

**Review by Nagashree01 — APPROVED** [review]
You can host this on netlify and give a preview link in description

**Pull request merged**